### DISCUSSION TITLE
automatic streaming does not work

### DISCUSSION THREAD

**Discussion by elitewieczorek** [WarmUpTill/SceneSwitcher#446] [Q&A] [answered]
Hi,
For a long time I was wondering how I can set my streams to turn on by themselves without my interference.
For example, to have the stream come on at 3 p.m. while I'm still at work.
It is true that I found the answer to this (Advanced Scene Switcher Plugin)
And okay, many people praise this plug, but I do not know how to set it.
I marked the stream to turn on in a minute, but the stream started a second after the button was pressed.
What am I doing wrong?
In A.S.S. 1.17.15 I have 4 tabs:
General / Macro / Network / Scene group
There are two tables in the macro:
in upper gives: If Streaming For exactly 1 minuts Stream starting
and at the bottom: Streaming> Start streaming

**Comment by WarmUpTill** [comment, marked as answer]
>For example, to have the stream come on at 3 p.m. while I'm still at work.

Have you tried something similar to this?

![Streaming](https://user-images.githubusercontent.com/19472752/164887526-c178a527-e8b0-4c40-b625-8ae981fec99f.PNG)

>in upper gives: If Streaming For exactly 1 minuts Stream starting
and at the bottom: Streaming> Start streaming

I don't understand what you are trying to achieve with the condition "If Streaming For exactly 1 minuts Stream starting".
Can you elaborate on why you choose these settings?

If you need some examples on how to use the plugin you can have a look at the [wiki](https://github.com/WarmUpTill/SceneSwitcher/wiki).

**Reply by elitewieczorek** [reply]
Thank you! Now works fine! I am sorry, I am Polish and I did not fully understand these options.
